WTWSMS/history/wars/isaurian_war.txt

```diff
@@ -1,19 +1,19 @@
-#name = "Isaurian Wars"
-#
-#casus_belli = {
-#	actor = 154001
-#	recipient = 70514
-#	casus_belli=tribal_invasion
-#	landed_title=k_asia
-#	date=492.1.1
-#}
-#
-#492.1.1 = {
-#	add_attacker = 154001
-#	add_defender = 70514
-#}
-#
-#497.5.5 = {
-#	rem_attacker = 154001
-#	rem_defender = 70514
-#}
+name = "Isaurian War"
```

Comment:
Here it is a question of preference, but "Wars" fit no?

Reply:
I changed it to "War" because it was a single conflict and this name seems mokre commonly used.

Reply:
Have you read the diary? https://forum.paradoxplaza.com/forum/index.php?threads/when-the-world-stopped-making-sense-a-480-migrational-period-mod.772971/page-169#post-17618762 I think the point is that it was a series of conflicts with revolts back and forth.

Reply:
I think I have, but a long time ago. Pretty interesting, though.

Everywhere this conflict mentioned I see "war" in singular (in different languages too).

Reply:
off-topic: This dev diary reminded me how much I miss the good old Byzzie colour. I tihnk I will make a tiny graphical submod to bring it back.

Reply:
As long as it is distributed separately that is fine. But you are speaking about a specific instance of the conflict, not the series of wars that happened. Hence the plural.

Reply:
I'm talking about the revolt (492-497), which was a single conflict, no?

Reply:
There were riots and then a lot of ping-pong within the empire against the Isaurians. All in all, it is more a question of what you define as a conflict or not, but I personally think status quo can stay here.

As for the colour, the reason for why I say it should be distributed separately is that I believe the mod players votes this change with Erik, so it makes little sense to include it as a sub-mod in the main download, especially since it is such a minor aspect.

Reply:
Yes, I would post it in a separate thread ofc.

Reply:
The inital open conflict and the later defensive strategy of the Isaurians were just stages of the same conflict, not separate conflicts.

Reply:
This was not the sole conflict with Isaurians however, hence the plural "wars".

Reply:
What do you mean? So there were peace periods between them?
Where have you found that plural form BTW? I have searched everywhere but literally all sites mention Isaurian War 😛
But if you think Wars are better, I guess they can stay